src/pages/subjects/Subjects.tsx

```
@@ -133,7 +129,8 @@
       axiosProps={{ onResponse: onResponseCategory }}
       labelField='name'
       onChange={onCategoryChange}
-      service={categoryService.getCategoriesNames}
+       // @ts-ignore
```

ESLint

[error] Delete `·`

(prettier/prettier)